Updated Polish (pl) localization of Appstores

piotrdrag opened the pull request:
@splewako

piotrdrag committed:
Updated Polish (pl) localization of Appstores

splewako commented:
Looks good but I wouldn't use all uppercase titles as it looks weird and wouldn't be consistent with other headers/"titles", 
https://play.google.com/store/apps/details?id=org.mozilla.firefox&hl=pl
https://play.google.com/store/apps/details?id=org.mozilla.firefox_beta&hl=pl

I would also start "twoim" (two occurrences) and "twój" (one) uppercase.

piotrdrag commented:
> Looks good but I wouldn't use all uppercase titles as it looks weird and wouldn't be consistent with other headers/"titles", 

You mean FAST, SMART, SAFE? They are not bold nor bulleted, so I think we need to preserve the upper case to indicate that they are titles, and not just extremely short sentences/paragraphs.

I've changed ABOUT MOZILLA to lower case, though.

> I would also start "twoim" (two occurrences) and "twój" (one) uppercase.

Fixed.

splewako commented:
> You mean FAST, SMART, SAFE? They are not bold nor bulleted, so I think we need to preserve the upper case to indicate that they are titles, and not just extremely short sentences/paragraphs.
> 
> I've changed ABOUT MOZILLA to lower case, though.

Yes, those. Intention is clear but it also looks like the person making decision didn't check them on site.

Landed, https://github.com/mozilla-l10n/appstores/commit/ee43c89199224eed3b6e1176b500d8cd32ab3ff9

Thank you for the patch
